website/README.md

```diff
@@ -84,32 +108,34 @@ git clone git@github.com:jordanpenard/brew-monitor.git .
 Install the required python libraries
 ```
 cd website
-pip3.7 install mod_wsgi
-pip3.7 install -r requirements.txt
+pip3.9 install mod_wsgi
+pip3.9 install -r requirements.txt
 ```
+Note: mod_wsgi is not in the `requirements.txt` because it requires apache to be installed. But maybe we should add it to ensure people have apache installed?
 
-Create /usr/local/brew-monitor/config/brew-monitor.wsgi
+Create the WSGI script: `/usr/local/brew-monitor/config/brew-monitor.wsgi`
 ```python
-#!/usr/local/bin/python3.7
+#!/usr/local/bin/python3.9
 
 import os
-os.environ['BREWMONITOR_CONFIG'] = '/usr/local/brew-monitor/config/config.yaml'
-
 import sys
+
+# Or use an actual env variable from your setup.
+os.environ['BREWMONITOR_CONFIG'] = '/usr/local/brew-monitor/config/config.yaml'
```

Comment:
Maybe we can do an if not def, then we set it to that fixed path ?

Reply:
No, this is an example file. In real deployment this should be in the environment not done in python as per the comment above.

For our own deployment you should maybe add a file in `/etc/profile.d/brewmonitor.sh` or `/etc/environment.d/99_brewmonitor.conf`.

Reply:
For example real scripts `make_dummy_data.py` does:
```
cfg_file = os.environ.get('BREWMONITOR_CONFIG', './debug_config.yaml')
```